cooltools/cli/call_dots.py

```diff
@@ -1,172 +1,184 @@
 import os.path as op
-from scipy.stats import poisson
 import pandas as pd
 import numpy as np
 import cooler
-import tempfile
 
 import click
 from . import cli
 from .. import dotfinder
 
 
-
 @cli.command()
 @click.argument(
     "cool_path",
     metavar="COOL_PATH",
-    type=str, #click.Path(exists=True, dir_okay=False),
-    nargs=1)
+    type=str,  # click.Path(exists=True, dir_okay=False),
+    nargs=1,
+)
 @click.argument(
     "expected_path",
     metavar="EXPECTED_PATH",
     type=click.Path(exists=True, dir_okay=False),
-    nargs=1)
+    nargs=1,
+)
 @click.option(
-    '--expected-name',
+    "--expected-name",
     help="Name of value column in EXPECTED_PATH",
     type=str,
-    default='balanced.avg',
+    default="balanced.avg",
     show_default=True,
-    )
+)
 @click.option(
-    '--balancing-weight-name',
+    "--balancing-weight-name",
```

Comment:
somehow this one was reverted - i've made it to be `--weight-name`  just before - now it's back to `--balancing...`
did you start of the latest version with the black-tools ?

Reply:
yes, i stupidly didn't merge your PR before making this branch!

Reply:
did you do a lot of manual editing after all? i'm wondering what's easier to fix

Reply:
resolve conflicts or start over with black/etc

Reply:
i ended up doing a bit of both